terraform/modules/services/airflow/s3.tf

```diff
@@ -47,21 +47,3 @@ resource "aws_iam_policy" "read-policy" {
   })
 }
 
-
-# resource "aws_s3_object" "dags" {
```

Comment:
Yea, lets pause and discuss perhaps different ways to do development and production deployments. We should make it easy to deploy, easy to development, easy to roll back if new version is bad, deploy seperately versions on dev and production envs